setup.py

```diff
@@ -5,38 +5,41 @@
 except ImportError:
     from distutils.core import setup
 
-
 install_reqs = ['cached-property',
```

Comment:
Can we add `backports.statistics` here? Pretty sure it should be installable for any version/system. Still good to have the manual fallback that you added, but it would be nice to force the package to install this I would think.